htdocs/holiday/class/holiday.class.php

```diff
@@ -22,1156 +19,1331 @@
+		'nb_open_day'		=> array('type' => 'double(24,8)', 'label' => 'NbUseDaysCPShort', 'enabled' => 1, 'position' => 75, 'notnull' => 0, 'visible' => 1, 'isameasure' => 1,),
+		'statut'			=> array('type' => 'integer', 'label' => 'Status', 'enabled' => 1, 'visible' => 1, 'notnull' => 1, 'default' => 0, 'index' => 1, 'position' => 2000, 'csslist' => 'center', 'arrayofkeyval' => array(1 => 'DraftCP', 2 => 'ToReviewCP', 3 => 'ApprovedCP', 4 => 'CancelCP', 5 => 'RefuseCP'), 'validate' => 1),
+		'date_valid'		=> array('type' => 'datetime', 'label' => 'DateValidation', 'enabled' => 1, 'position' => 85, 'notnull' => 0, 'visible' => 1,),
+		'fk_user_valid'		=> array('type' => 'integer:User:user/class/user.class.php', 'label' => 'UserValidation', 'enabled' => 1, 'position' => 86, 'notnull' => 0, 'visible' => -1, 'css' => 'maxwidth500 widthcentpercentminusxx', 'csslist' => 'tdoverflowmax150',),
+		'date_approval'		=> array('type' => 'datetime', 'label' => 'DateApprove', 'enabled' => 1, 'position' => 90, 'notnull' => 0, 'visible' => 1,),
+		'fk_user_approve'	=> array('type' => 'integer:User:user/class/user.class.php', 'label' => 'Fkuserapprove', 'enabled' => 1, 'position' => 91, 'notnull' => 0, 'visible' => -1, 'css' => 'maxwidth500 widthcentpercentminusxx', 'csslist' => 'tdoverflowmax150',),
+		'date_refuse'		=> array('type' => 'datetime', 'label' => 'Daterefuse', 'enabled' => 1, 'position' => 95, 'notnull' => 0, 'visible' => -1,),
+		'fk_user_refuse'	=> array('type' => 'integer:User:user/class/user.class.php', 'label' => 'Fkuserrefuse', 'enabled' => 1, 'position' => 96, 'notnull' => 0, 'visible' => -1, 'css' => 'maxwidth500 widthcentpercentminusxx', 'csslist' => 'tdoverflowmax150',),
+		'date_cancel'		=> array('type' => 'datetime', 'label' => 'Datecancel', 'enabled' => 1, 'position' => 105, 'notnull' => 0, 'visible' => -1,),
+		'fk_user_cancel'	=> array('type' => 'integer:User:user/class/user.class.php', 'label' => 'Fkusercancel', 'enabled' => 1, 'position' => 110, 'notnull' => 0, 'visible' => -1, 'css' => 'maxwidth500 widthcentpercentminusxx', 'csslist' => 'tdoverflowmax150',),
+		'detail_refuse'		=> array('type' => 'varchar(250)', 'label' => 'DetailRefuse', 'enabled' => 1, 'position' => 115, 'notnull' => 0, 'visible' => -1,),
+		'note_public'		=> array('type' => 'html', 'label' => 'NotePublic', 'enabled' => 1, 'visible' => 0, 'position' => 56, 'validate' => 1, 'cssview' => 'wordbreak'),
+		'note_private'		=> array('type' => 'html', 'label' => 'NotePrivate', 'enabled' => 1, 'visible' => 0, 'position' => 57, 'validate' => 1, 'cssview' => 'wordbreak'),
+		'tms'				=> array('type' => 'timestamp', 'label' => 'DateModification', 'enabled' => 1, 'visible' => -2, 'notnull' => 0, 'position' => 51),
+		'import_key'		=> array('type' => 'varchar(14)', 'label' => 'ImportId', 'enabled' => 1, 'visible' => -2, 'notnull' => -1, 'index' => 0, 'position' => 1000),
+	);
+	public $rowid;
```

Comment:
Missing comment to defined type
	/**
	 * @var int ID
	 */

Reply:
Won't this be overwritten by modulebuilder ? It's in the "MODULEBUILDER PROPERTIES" auto generated section